docs/usecases/external_collaboration.rst

```diff
@@ -0,0 +1,364 @@
+          dfs.append(df)
+
+      combined_data = pd.concat(dfs, ignore_index=True)
+
+      # read local samples
+      local_samples = pd.read_csv(measurements,
+								usecols=['Culmen Length (mm)', 'Culmen Depth (mm)'])
+
+   Next, it builds the model using variable names from the spreadsheet:
+   "Culmen Length (mm)" and "Culmen Depth (mm)" are used as features to predict the "Species".
+   The model performance is evaluated in a cross-validation, and the script prints the average accuracy.
+
+   .. code-block::
+
+      # define features and targets, build a smaller dataset
+      penguins = combined_data[["Culmen Length (mm)", "Culmen Depth (mm)", "Species"]]
+      penguins = penguins.dropna()
+      data, target = penguins.drop(columns="Species"), penguins["Species"]
+
+
+      # build a pipeline with a Logistic Regression
+      model = make_pipeline(StandardScaler(), LogisticRegression())
+      # evaluate the model using cross-validation
+      cv_result = cross_validate(model, data, target, cv=3)
+      print(f'average model accuracy is {cv_result["test_score"].mean():.3f}')
+
+   Finally, the trained model is used to predict the penguin species of your own samples, and writes it to a file.
+
+   .. code-block::
+
+      # fit the model on data; predict Species of own data points
+      model.fit(data, target)
+      res = model.predict(local_samples)
+      local_samples['prediction'] = res
+      # save the prediction
+      local_samples.to_csv(args.output)
+
+
+To make things easier for the data provider, you can add a software container that includes all required software, using the ``datalad-containers`` extension.
+
+.. runrecord:: _examples/remote-analysis-105
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad containers-add software --url shub://adswa/resources:2
+
+Let's run the script on mock data in the software container.
+The results will not be correct because the data is only simulated, but it generates a run record that the data provider can very easily rerun - sparing you the need for detailed instructions in an Email or a README file.
+Keep in mind to properly define inputs and outputs, so that the analysis can easily be rerun.
+
+.. runrecord:: _examples/remote-analysis-106
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ datalad containers-run -n software -m "run analysis on mock data" -i 'inputs/*/*table*.csv' -o "predictions.csv" "python3 code/predict.py"
+
+Note how bad the model performs - this accuracy will likely improve on the real data.
+We can tag the dataset state to make rerunning easier:
+
+.. runrecord:: _examples/remote-analysis-107
+   :language: console
+   :workdir: usecases/remote-analysis/penguin-jelly
+
+   $ git tag runme
+
+The question is: How does this look like on the real data?
+A data provider would clone your dataset and replace the subdataset with the actual sensitive dataset.
+
+We can pretend to do this:
```

Comment:
I understand why the clone of the external collaborator is reused here. But I think this should not be done, because it makes it hard to see where exactly the line between provider and consumer is.

I think the `drop` should go and this part should start with a section heading (Provider perspective) and a `clone` from the source where the prepared YODA dataset can be found. This could either be the one prepared in the code above, or a dedicated "edu" resource. Given that we already have to have `empty` and `mock`, we can also have `jelly` hosted there.

This would make the message "put it somewhere accessible for the provider".

Reply:
OK, I now see that this section heading comes next already.

I find this confusing. The rerun demo here appears to communicate the provider perspective to the consumer. But the next section communicates the provider perspective to any reader. Hence this feels duplicate and adds unnecessary complexity.

Reply:
addressed by reordering as you proposed.